src/components/ui/calendar.tsx

```diff
@@ -40,22 +24,22 @@ function Calendar({
         month: "space-y-4",
         caption: "flex justify-center pt-1 relative items-center",
         caption_label: "text-sm font-medium",
-        // nav: "space-x-1 flex items-center",
+        nav: "space-x-1 flex items-center",
         nav_button: cn(
           buttonVariants({ variant: "outline" }),
           "h-7 w-7 bg-transparent p-0 opacity-50 hover:opacity-100"
         ),
         nav_button_previous: "absolute left-1",
         nav_button_next: "absolute right-1",
         table: "w-full border-collapse space-y-1",
-        head_row: "flex w-full",
+        head_row: "flex",
         head_cell:
-          "flex-1 text-center text-muted-foreground rounded-md w-9 font-normal text-[0.8rem]",
+          "text-muted-foreground rounded-md w-9 font-normal text-[0.8rem]",
         row: "flex w-full mt-2",
-        day: "flex-1 text-center text-sm p-0 relative [&:has([aria-selected].day-range-end)]:rounded-r-md [&:has([aria-selected].day-outside)]:bg-accent/50 [&:has([aria-selected])]:bg-accent first:[&:has([aria-selected])]:rounded-l-md last:[&:has([aria-selected])]:rounded-r-md focus-within:relative focus-within:z-20",
-        day_button: cn(
+        cell: "h-9 w-9 text-center text-sm p-0 relative [&:has([aria-selected].day-range-end)]:rounded-r-md [&:has([aria-selected].day-outside)]:bg-accent/50 [&:has([aria-selected])]:bg-accent first:[&:has([aria-selected])]:rounded-l-md last:[&:has([aria-selected])]:rounded-r-md focus-within:relative focus-within:z-20",
+        day: cn(
           buttonVariants({ variant: "ghost" }),
-          "h-9 w-9 p-0 font-normal aria-selected:opacity-100 mx-auto"
+          "h-9 w-9 p-0 font-normal aria-selected:opacity-100"
```

Comment:
I had these custom changes in here because the Calendar component doesn't quite render correctly when using React@19 and the other dependencies it relies on.

<img width="644" alt="Screenshot 2025-02-16 at 12 19 48" src="https://github.com/user-attachments/assets/fe849369-9e28-40ea-8d9d-1c1fd5557abb" />

See what I mean? The day labels above the calendar aren't lined up, the <> arrows aren't where they should be, etc.

We can talk about the best way to handle this if you're unsure.

Reply:
oh I see that yeah. These were updated automatically when I updated the calendar component using the CLI. Let's chat about it yeah

Reply:
If you leave the style changes in place and the temporary comment that I put in there, it should fix it well enough to move on. There's more we should do, and probably need to reach out to shadcn about this, but it's what I could figure out when working on this problem around Christmas time.